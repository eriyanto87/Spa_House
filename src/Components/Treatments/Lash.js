import React from "react";
import { Link } from "react-router-dom";

export default function Lash() {
  return (
    <div>
      <h3>Lash</h3>
      <div className="services">
        <h4>Classic</h4>
        <p>
          A technique in which a single extension is applied to one, isolated
          natural lash.
        </p>
        <Link to={{ pathname: `/pricing/lash` }}>
          <button>Next Step</button>
        </Link>{" "}
      </div>
    </div>
  );
}
