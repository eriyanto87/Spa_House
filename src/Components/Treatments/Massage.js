import React from "react";
import { Link } from "react-router-dom";

export default function Massage(props) {
  return (
    <div>
      <section className="service">
        <div className="services">
          <h4>Relaxation</h4>
          <p>
            Long, gliding, flowing strokes to calm the mind and body, increase
            circulation, and promote deep relaxation.
          </p>
          <Link to={{ pathname: `/pricing` }}>
            <button>Next Step</button>
          </Link>{" "}
        </div>
        <div className="services">
          <h4>Deep Tissue</h4>
          <p>
            Targeted, firm pressure to reach the deepest layers of muscle,
            fascia, and tendons.
          </p>
          <Link to={{ pathname: `/pricing` }}>
            <button>Next Step</button>
          </Link>{" "}
        </div>
        <div className="services">
          <h4>Thai</h4>
          <p>
            Traditional healing system combining acupressure and assisted
            stretching. The recipient wears loose, comfortable clothing. The
            service will be done on a firm mat.{" "}
          </p>
          <Link to={{ pathname: `/pricing` }}>
            <button>Next Step</button>
          </Link>{" "}
        </div>
        <div className="services">
          <h4>Prenatal/Pregnancy</h4>
          <p>
            You will be positioned lying on your side rather than on your
            stomach or back. Pillows, bolsters, or padding may be used to
            support your back, knees, and/or feet.{" "}
          </p>
          <Link to={{ pathname: `/pricing` }}>
            <button>Next Step</button>
          </Link>{" "}
        </div>
      </section>
    </div>
  );
}
