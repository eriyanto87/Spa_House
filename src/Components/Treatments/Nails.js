import React from "react";
import { Link } from "react-router-dom";

export default function Nails() {
  return (
    <div>
      <h3>Nails</h3>
      <div className="services">
        <h4>Manicure</h4>
        <p></p>
        <Link to={{ pathname: `/pricing/massage` }}>
          <button>Next Step</button>
        </Link>{" "}
      </div>
      <div className="services">
        <h4>Pedicure</h4>
        <p></p>
        <Link to={{ pathname: `/pricing/massage` }}>
          <button>Next Step</button>
        </Link>{" "}
      </div>
      <div className="services">
        <h4>Gel Manicure</h4>
        <p></p>
        <Link to={{ pathname: `/pricing/massage` }}>
          <button>Next Step</button>
        </Link>{" "}
      </div>
    </div>
  );
}
