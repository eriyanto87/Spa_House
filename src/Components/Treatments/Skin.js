import React from "react";
import { Link } from "react-router-dom";

export default function Skin() {
  return (
    <div>
      <h3>Skin</h3>
      <div className="services">
        <h4>Facials</h4>
        <p></p>
        <Link to={{ pathname: `/pricing` }}>
          <button>Next Step</button>
        </Link>{" "}
      </div>
      <div className="services">
        <h4>Waxing</h4>
        <p></p>
        <Link to={{ pathname: `/pricing` }}>
          <button>Next Step</button>
        </Link>{" "}
      </div>
    </div>
  );
}
