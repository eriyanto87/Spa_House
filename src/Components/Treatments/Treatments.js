import React from "react";
import Massage from "./Massage";
import Skin from "./Skin";
import Lash from "./Lash";
import Nails from "./Nails";
import Nav from "../Nav/Nav";
import Footer from "../Footer/Footer";

export default function Treatments() {
  const [state, setState] = React.useState("massage");

  return (
    <div>
      <Nav />
      <section>
        <h3>Treatments</h3>
        <select
          onChange={(e) => {
            setState(e.target.value);
          }}
          name="treatments"
          id="treatment"
        >
          <option value="massage">Massage</option>
          <option value="skincare">Skin Care</option>
          <option value="nailcare">Nail Care</option>
          <option value="lash">Eyelash Extensions</option>
        </select>
        {state === "massage" && <Massage />}
        {state === "skincare" && <Skin />}
        {state === "nailcare" && <Nails />}
        {state === "lash" && <Lash />}
      </section>
      <Footer />
    </div>
  );
}
