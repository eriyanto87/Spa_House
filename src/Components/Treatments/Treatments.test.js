import React from "react";
import ReactDOM from "react-dom";
import { BrowserRouter } from "react-router-dom";
import renderer from "react-test-renderer";
import Treatments from "./Treatments";

describe("Treatments component", () => {
  it("Treatmest Snapshot", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Treatments />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(
      <BrowserRouter>
        <Treatments />
      </BrowserRouter>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });
});
