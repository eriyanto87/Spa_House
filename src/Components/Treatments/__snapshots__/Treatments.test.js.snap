// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Treatments component Treatmest Snapshot 1`] = `
<div>
  <div
    className="nav"
  >
    <ul>
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
          Home 
        </a>
      </li>
      <li>
        <a
          href="/about"
          onClick={[Function]}
        >
          About 
        </a>
      </li>
      <li>
        <a
          href="/treatments"
          onClick={[Function]}
        >
          Treatments 
        </a>
      </li>
      <li>
        <a
          href="/pricing"
          onClick={[Function]}
        >
          Pricing 
        </a>
      </li>
      <li>
        <a
          href="/contact"
          onClick={[Function]}
        >
          Contact Us 
        </a>
      </li>
      <li>
        <a
          href="/gallery"
          onClick={[Function]}
        >
          Gallery
        </a>
      </li>
    </ul>
  </div>
  <section>
    <h3>
      Treatments
    </h3>
    <select
      id="treatment"
      name="treatments"
      onChange={[Function]}
    >
      <option
        value="massage"
      >
        Massage
      </option>
      <option
        value="skincare"
      >
        Skin Care
      </option>
      <option
        value="nailcare"
      >
        Nail Care
      </option>
      <option
        value="lash"
      >
        Eyelash Extensions
      </option>
    </select>
    <div>
      <section
        className="service"
      >
        <div
          className="services"
        >
          <h4>
            Relaxation
          </h4>
          <p>
            Long, gliding, flowing strokes to calm the mind and body, increase circulation, and promote deep relaxation.
          </p>
          <a
            href="/pricing"
            onClick={[Function]}
          >
            <button>
              Next Step
            </button>
          </a>
           
        </div>
        <div
          className="services"
        >
          <h4>
            Deep Tissue
          </h4>
          <p>
            Targeted, firm pressure to reach the deepest layers of muscle, fascia, and tendons.
          </p>
          <a
            href="/pricing"
            onClick={[Function]}
          >
            <button>
              Next Step
            </button>
          </a>
           
        </div>
        <div
          className="services"
        >
          <h4>
            Thai
          </h4>
          <p>
            Traditional healing system combining acupressure and assisted stretching. The recipient wears loose, comfortable clothing. The service will be done on a firm mat.
             
          </p>
          <a
            href="/pricing"
            onClick={[Function]}
          >
            <button>
              Next Step
            </button>
          </a>
           
        </div>
        <div
          className="serviceslast"
        >
          <h4>
            Prenatal/Pregnancy
          </h4>
          <p>
            You will be positioned lying on your side rather than on your stomach or back. Pillows, bolsters, or padding may be used to support your back, knees, and/or feet.
             
          </p>
          <a
            href="/pricing"
            onClick={[Function]}
          >
            <button>
              Next Step
            </button>
          </a>
           
        </div>
      </section>
    </div>
  </section>
  <div>
    <p>
      Â© 2020
    </p>
  </div>
</div>
`;
